packages/core/src/common/classes.ts

```diff
@@ -116,8 +116,14 @@ export const CONTROL_GROUP = `${NS}-control-group`;
 export const DIALOG = `${NS}-dialog`;
 export const DIALOG_CONTAINER = `${DIALOG}-container`;
 export const DIALOG_BODY = `${DIALOG}-body`;
+export const DIALOG_BODY_NO_PADDING = `${DIALOG}-body-no-padding`;
```

Comment:
<DialogBody /> not only provides padding but also scroll. Thus, I'm adding a way to remove padding as just not using <DialogBody /> is not practical.
Adding a "subtractive" class to keep DIALOG_BODY as close from current.